tests/manual/manual_err.rs

```diff
@@ -171,3 +204,32 @@ fn test_other_err_fixer() {
     }
     ");
 }
+
+#[test]
+fn test_basic_err_block_diagnostics() {
+    test_lint_diagnostics!(TEST_BASIC_ERR_BLOCK, @r"
+    Plugin diagnostic: Manual match for `err` detected. Consider using `err()` instead
+     --> lib.cairo:5:16-14:5
+          let _foo = match foo {
+     ________________^
+    | ...
+    |     };
+    |_____^
+    ");
+}
+
+#[test]
+fn test_basic_err_block_fixer() {
+    test_lint_fixer!(TEST_BASIC_ERR_BLOCK, @r"
+    fn main() {
+        let foo: Result<i32> = Result::Err('err');
+        // This is just a variable.
+        let _foo = foo.err();
+    }
+    ");
+}
+
+#[test]
+fn test_basic_err_block_with_more_statements_diagnostics() {
```

Comment:
nit: please add a fixer test. I know it won't modify the code, but we would make sure nothing is touched here as it's supposed to be